node/network/collator-protocol/src/collator_side.rs

```diff
@@ -1201,8 +1198,8 @@ mod tests {
 	async fn distribute_collation(
 		virtual_overseer: &mut VirtualOverseer,
 		test_state: &TestState,
-		// whether or not the currently active validators are already connected or not.
-		already_connected: bool,
+		// whether or not we expect a connection request or not.
+		should_connect: bool,
```

Comment:
is it ever false? looks like we can remove the flag entirely

Reply:
Once we finished our improvements to validator discovery we are going to need it again. So I would rather not remove it, to just add it back again a PR later.